### PR TITLE
Deployment command change

### PR DESCRIPTION
**Pull request by Renu05**
setup:static:deploy to setup:static-content:deploy

## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
whilst this is a correct correction, documentation should no longer use "php magento"

As this file in PR, please can you change all instances of "php magento" to "bin/magento"

eg the line changed should read : bin/magento setup:static-content:deploy

Please also complete the PR description full - see WIKI

**Review by keharper — APPROVED** [review]

**Review by keharper — CHANGES_REQUESTED** [review]
Approved too early. Requesting changes

**Comment by Renu05** [comment]
Hi @keharper ,
I have made the changes magento to bin/magento

File: src/cloud/before/before-setup-env-install.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ To install Magento using the command line:
 1. Enter a CLI command with options for entering the name, email, ADMIN credentials, URL, and additional information. For a list of all options, see [Installer help commands]({{ site.baseurl }}/guides/v2.3/install-gde/install/cli/install-cli-install.html#instgde-cli-help-cmds).
 
    ```bash
-   php magento setup:install \
+   php bin/magento setup:install \
```
Comment (dobooth) [inline src/cloud/before/before-setup-env-install.md:126]:
```suggestion
   bin/magento setup:install \
```

We not longer need `php` at the beginning of commands. Please remove the other two as well.

**Review by dobooth — CHANGES_REQUESTED** [review]

**Comment by dobooth** [comment]
Closing as abandoned.

**Pull request closed without merging**